components/wpa_supplicant/src/tls/rsa.c

```diff
@@ -13,41 +13,198 @@
 #include "tls/bignum.h"
 #include "tls/rsa.h"
 
+#ifdef SE050_ENABLED
+#include "libtommath.h"
+#include <ex_sss.h>
+#include <ex_sss_auth.h>
+#include <ex_sss_boot.h>
+#include <fsl_sss_se05x_apis.h>
+#include <nxEnsure.h>
+#include <nxLog_App.h>
+#include <se05x_APDU_apis.h>
+#include <se05x_enums.h>
+
+#define EX_SSS_BOOT_PCONTEXT (&gex_sss_get_timestamp_boot_ctx)
+//#define EX_SSS_BOOT_DO_ERASE 1
+#define EX_SSS_BOOT_EXPOSE_ARGC_ARGV 0
+//#define DIGEST_LENGTH 32
+
+uint32_t se050_obj_id = 0;
+uint8_t item[256];
+uint32_t p1 = 0;
+uint32_t coef = 0;
+
+static ex_sss_boot_ctx_t gex_sss_get_timestamp_boot_ctx;
+
+sss_status_t status;
+extern uint8_t signature[512] = {0};
+size_t signatureLen=0;
+sss_algorithm_t algorithm;
+sss_mode_t mode;
+size_t keylen=0;
+
+sss_object_t key;
+sss_asymmetric_t ctx_asymm = {0};
+uint32_t keyId = 0x0; //Secure Element ID hosting the RSA Injected Key
+#endif
+
 struct crypto_rsa_key {
-	int private_key; /* whether private key is set */
-	struct bignum *n; /* modulus (p * q) */
-	struct bignum *e; /* public exponent */
-	/* The following parameters are available only if private_key is set */
-	struct bignum *d; /* private exponent */
-	struct bignum *p; /* prime p (factor of n) */
-	struct bignum *q; /* prime q (factor of n) */
-	struct bignum *dmp1; /* d mod (p - 1); CRT exponent */
-	struct bignum *dmq1; /* d mod (q - 1); CRT exponent */
-	struct bignum *iqmp; /* 1 / q mod p; CRT coefficient */
+    int private_key; /* whether private key is set */
```

Comment:
no whitespace *changes* please unless you have a very specific reason, but in IDF we should really not change existing code unless we must